packages/samples/headless-react/src/components/category-facet/category-facet-search.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  CategoryFacetSearch as HeadlessCategoryFacetSearch,
+  CategoryFacetSearchState,
+} from '@coveo/headless';
+import {FunctionComponent} from 'react';
+
+export interface CategoryFacetSearchProps {
+  controller: HeadlessCategoryFacetSearch;
+  facetState: CategoryFacetSearchState;
+}
+
+export const CategoryFacetSearch: FunctionComponent<CategoryFacetSearchProps> = (
+  props
+) => {
+  const onInput = (text: string) => {
+    props.controller.updateText(text);
+    props.controller.search();
+  };
+
+  return (
+    <div>
+      <input onInput={(e) => onInput(e.currentTarget.value)} />
+      <ul>
+        {props.facetState.values.map((value) => (
+          <li key={[...value.path, value.rawValue].join('>')}>
```

Comment:
This is probably something we should improve in headless eventually. The `path` doesn't include the current value in `CategoryFacetSearchState`, but it does in `CategoryFacetValue`.

Reply:
Is this inconsistency coming from headless? or the facet search api response?

If it's at the api level, then it might be better to contact the team so that all consumers of the api get a consistent behaviour.

Reply:
It comes from the search api response.